zero/src/rpc/jumpdest.rs

```diff
@@ -0,0 +1,512 @@
+use core::default::Default;
+use core::option::Option::None;
+use core::str::FromStr as _;
+use core::time::Duration;
+use std::collections::BTreeMap;
+use std::collections::HashMap;
+use std::ops::Not as _;
+
+use __compat_primitive_types::H160;
+use __compat_primitive_types::H256;
```

Comment:
These are named weirdly to _discourage_ use - can you use the `alloy` types instead?